### PR TITLE
Create builder from existing json

### PR DESCRIPTION
**Pull request by bjorncs**
I confirm that this contribution is made under the terms of the license found in the root directory of this repository's source tree and that I have the authority necessary to make this contribution on behalf of its copyright owner.

### CODE REVIEW

File: vespajlib/src/main/java/ai/vespa/json/Json.java [hunk 1]
```diff
@@ -211,6 +211,18 @@ public static Builder.Array existingSlimeArrayCursor(Cursor cursor) {
             if (cursor.type() != Type.ARRAY) throw new InvalidJsonException("Input is not an array");
             return new Builder.Array(cursor);
         }
+        public static Builder.Object fromObject(Json json) {
```
Comment (hakonhall) [inline vespajlib/src/main/java/ai/vespa/json/Json.java:214]: There should be a from(Json) too, that delegates to one of Object or Array depending on the type?

**Review by hakonhall — APPROVED** [review]

File: vespajlib/src/main/java/ai/vespa/json/Json.java (hunk 1 above)
Comment (bjorncs) [reply vespajlib/src/main/java/ai/vespa/json/Json.java:214]: That method would have the return type `Builder` which does not provide any methods for mutating the structure.

Comment (hakonhall) [reply vespajlib/src/main/java/ai/vespa/json/Json.java:214]: makes sense

Comment (bjorncs) [reply vespajlib/src/main/java/ai/vespa/json/Json.java:214]: @jonmv `json.requireType(Type.OBJECT)` will throw `InvalidJsonException`

Comment (jonmv) [reply vespajlib/src/main/java/ai/vespa/json/Json.java:214]: Yes ... >_<

**Pull request merged**